velox/docs/functions/spark/datetime.rst

```diff
@@ -29,13 +29,16 @@ These functions support TIMESTAMP and DATE input types.
     If num_days is a negative value then these amount of days will be
     deducted from start_date.
 
-.. spark:function:: datediff(endDate, startDate) -> integer
+.. spark:function:: date_format(timestamp, format) -> string
 
-    Returns the number of days from startDate to endDate. Only DATE type is allowed
-    for input. ::
+    Converts ``timestamp`` to a date/time string (local timezone adjusted) in the format specified by
```

Comment:
What does 'local timezone adjusted' mean? Would you clarify for the reader?

Reply:
It meant the time should be adjusted to configured session timezone. Just revised. Thanks!